css/css-ui/appearance-button-drop-down-select-001.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: appearance: button on drop-down select</title>
+<link rel="help" href="https://drafts.csswg.org/css-ui-4/#appearance-switching">
+<meta name="assert" content="The button value works on a drop-down select.">
+<link rel="mismatch" href="drop-down-select-ref.html">
```

Comment:
There's nothing that requires that the look and feel of <select> and the look and feel of <button> be the same, so the reference file should contain an actual button element, otherwise the test is not doing what it's claiming.

Reply:
Note that the reference is "mismatch", i.e. they should not match. An actual button might have different size than a select with appearance: button (e.g. due to different default border width), so it doesn't really work to have a "match" reference here I think. (Setting the border with author CSS disables the appearance, so we can't do that, either.)